src/main/java/com/mycompany/hangman/repo/AbstractRepository.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.mycompany.hangman.repo;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *
+ * @author Cory
+ * @param <T>
+ */
+public abstract class AbstractRepository<T> implements Repository<T>

```

Comment:
i’d call this an “InMemory” repo. not sure if it makes sense to provide an abstract class with such a specific implementation. other types of repo subclasses would likely do completely different things like fetch/store from the web or from a database or from the windows registry. not sure they would have a use for this map member. i’d move this down to the Default repo and delete this abstract class. if you find a use for an abstract class in the future then you can add one back in that meets that usage